ee_tests/src/page_objects/user_settings.page.ts

```diff
@@ -11,10 +11,16 @@ export class UserSettingsPage extends AppPage {
 }
 
 export class FeaturesTab {
-
+    featureInputXpath = '//input/ancestor::*[contains(@class, \'active\')]//span[contains(@class,\'icon-\')]';
```

Comment:
You are using the variable without a declaration. Although we are not running tslint, we will in the future, let's keep the quality of our code high. Please add `let` or `const`.